src/components/Text/Text.jsx

```diff
@@ -1,68 +1,57 @@
 import styled from "styled-components";
 import PropType from "prop-types";
+import designTokens from "../../config/designTokens";
 
 const Text = styled.p`
   color: ${(props) => props.color};
-  font-size: ${(props) => props.size}px;
-  font-weight: ${(props) => props.weight};
-  font-family: ${(props) => props.family};
+  font-size: ${(props) => getFontSize(props.size)}px;
+  font-weight: ${(props) => getFontWeight(props.weight)};
+  font-family: ${(props) => getFontFamily(props.weight)};
   text-decoration: ${(props) => props.decoration};
-  line-height: ${(props) => props.height}px;
+  line-height: ${(props) => getLineHeight(props.size)}px;
   text-align: ${(props) => props.align};
-  @media screen and (min-width: 640px) {
+  @media screen and (min-width: ${designTokens.breakpoints.tablet}px) {
     text-align: ${(props) => props.tabletAlign};
   }
-  @media screen and (min-width: 960px) {
+  @media screen and (min-width: ${designTokens.breakpoints.desktop}px) {
     text-align: ${(props) => props.desktopAlign};
   }
-
-  // This props standardizes text types if you don't want to customize
-  // and choose one already assembled. It is not yet configured with the
-  // right styles, other components need to be implemented first.
-  ${(props) => {
-    switch (props.type) {
-      case "default":
-        return "color: red; font-size: 50px; font-family: Quicksand, sans-serif; font-weight: 100";
-      case "default2":
-        return "color: green";
-      default:
-        return;
-    }
-  }}
 `;
 
+Text.defaultProps = {
+  family: "regular" | "medium" | "bold",
+  size: "big" | "default" | "small" | "xsmall",
+  weight: "regular" | "medium" | "bold",
+};
+
+const getFontWeight = (weight) => {
+  switch (weight) {
+    case "bold":
+      return 700;
+    case "medium":
+      return 500;
+    case "regular":
+      return 400;
+    default:
+      return null;
+  }
+};
+
+const getFontFamily = (weight) => designTokens.typography.text.family[weight];
+const getFontSize = (size) => designTokens.typography.text.sizes[size].size;
+const getLineHeight = (size) =>
+  designTokens.typography.text.sizes[size].lineHeight;
```

Comment:
Maybe this line should be on the line above, such as the others constants. But this it's just a :nail_care:
```suggestion
  const getLineHeight = (size) => designTokens.typography.text.sizes[size].lineHeight;
```

Reply:
Vini is just the formatting, when the const gets big the react itself breaks :)

Reply:
Ah, great! That's it then :slightly_smiling_face: